hacking/shippable/rebalance.py

```diff
@@ -22,11 +22,13 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
+import datetime
```

Comment:
Omit the changes to this script. The JSON files it used to rebalance targets are not currently saved as Azure Pipelines artifacts, so the necessary data is currently unavailable. Once that's resolved this script can be updated in a separate PR.